ensime-helm.el

```diff
@@ -9,9 +9,19 @@
 
 (require 'helm "helm.el" t)
 
+(defun string-or-empty (str)
```

Comment:
Functions should always have an ensime prefix. Elisp lacks namespaces. Maybe instead of this function `(or str "" )` is good enough.

Reply:
Thanks! Sounds like an excellent improvement :)